scripts/bump_version.py

```
@@ -0,0 +1,79 @@
+    _set_version(workspace, version)
+    for member in members:
+        member_path = root / member / "Cargo.toml"
+        if not member_path.exists():
+            print(
+                f"Warning: Skipping missing member Cargo.toml at {member_path}",
+                file=sys.stderr,
+            )
+            continue
+        try:
+            _set_version(member_path, version)
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)